09-intermediate-rails/reference/.rails-template.rb

```diff
@@ -49,33 +50,39 @@
   inject_into_file 'app/assets/javascripts/application.js', after: '// about supported directives.' do
     <<-'JAVASCRIPT'
 
-    //= require jquery
+  //= require jquery3
+  //= require popper
+  //= require bootstrap-sprockets
     JAVASCRIPT
   end
 
-  # Add normalize-rails
-  inject_into_file 'app/assets/stylesheets/application.css', after: ' * It is generally better to create a new file per style scope.' do
+  append_to_file 'app/assets/stylesheets/application.css' do
     <<-'SCSS'
 
-    *= require normalize-rails
+/* Custom bootstrap variables must be set or imported *before* bootstrap. */
+@import "bootstrap";
     SCSS
   end
 end
-
+gsub_file 'app/assets/stylesheets/application.css', / \*= require_tree .\n/ do
+  ""
+end
+gsub_file 'app/assets/stylesheets/application.css', / \*= require_self\n/ do
```

Comment:
I'm confused by the code to take out these two `require_whatever` directives. Is loading all of the other CSS files still working after these are removed?